corehq/apps/app_manager/models.py

```diff
@@ -3680,6 +3687,7 @@ class Application(ApplicationBase, TranslationMixin, HQMediaMixin):
                                           choices=app_strings.CHOICES.keys())
     commtrack_requisition_mode = StringProperty(choices=CT_REQUISITION_MODES)
     auto_gps_capture = BooleanProperty(default=False)
+    created_from = StringProperty()
```

Comment:
The thought on the spec had been that it might apply to apps created in other new ways, so sometimes it might be "template_survey" and other times "exchange_<app_id>" ...but yeah, parsing a prefix does seem annoying. Changed to "created_from_template"